### PR TITLE
Run acceptance tests in Travis against deployed server

### PR DESCRIPTION
**Pull request by mbdebbeler**
**Added:** 

- a /scripts folder

- a command to make the script files executable on the VM (permissions change)

**Changed:**

- Appspec.yml - renamed the script that starts the server and leveled up in the EC2 to make sure CodeDeploy starts the most-recently-built version of the .jar

- Travis.yml - separated the stages into: build, run unit tests, deploy, clone and run acceptance tests.

**Moved:**

- Scripts into the /scripts folder

- Script commands into different stages of the Travis build

**Removed:**

- Nothing.

### CODE REVIEW

File: appspec.yml [hunk 1]
```diff
@@ -3,11 +3,11 @@ os: linux
...
-  - source: deploy_server.sh
+  - source: /scripts/start_server.sh
     destination: /opt
   - source: /Logs
     destination: /opt
 hooks:
-  ApplicationStart:
-    - location: deploy_server.sh
+  AfterInstall:
+    - location: ../../../../../start_server.sh
```
Comment (Bjornkjohnson) [inline appspec.yml]: Any luck on the absolute path here?

File: scripts/clone_and_run_acceptance_tests.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -ex
+gem install bundler
+wget https://github.com/mbdebbeler/http_server_spec/archive/master.tar.gz
+tar -xvzf master.tar.gz
+cd http_server_spec-master && bundle install
+sed -i -e 's/127.0.0.1/ec2-3-15-197-208.us-east-2.compute.amazonaws.com/' ./features/support/config.yml
```
Comment (Bjornkjohnson) [inline scripts/clone_and_run_acceptance_tests.sh]: Don't forget to try moving this string to an env variable if possible.

File: appspec.yml (hunk 1 above)
Comment (mbdebbeler) [reply appspec.yml]: Just put these two changes on a separate branch - going to test and then will make a PR once it's working!

File: scripts/clone_and_run_acceptance_tests.sh (hunk 2 above)
Comment (mbdebbeler) [reply scripts/clone_and_run_acceptance_tests.sh]: Yep, testing now to make sure I can access the Travis ENV variables from the scripts.

**Pull request merged**